indra/llwindow/lldxhardware.cpp

```diff
@@ -1059,7 +1059,7 @@ LLSD LLDXHardware::getDisplayInfo()
 
         // Dump the string as an int into the structure
         char *stopstring;
-        ret["VRAM"] = strtol(ram_str.c_str(), &stopstring, 10);
+        ret["VRAM"] = (LLSD::Integer)strtol(ram_str.c_str(), &stopstring, 10);
```

Comment:
In general, I prefer C++ iniitalization to C-style casts, e.g.
```
LLSD::Integer(strtol(...))
```
If that complains, you could use `narrow()`:
```
LLSD::Integer(narrow(strtol(...)))
```
or ... you could consider using `stoi()` instead, which returns `int`.

Reply:
Fixed!

Reply:
Thanks!